src/modules/Client/Service.php

```diff
@@ -469,7 +469,7 @@ public function deleteGroup(\Model_ClientGroup $model)
     {
         $client = $this->di['db']->findOne('Client', 'client_group_id = ?', [$model->id]);
         if ($client) {
-            throw new \FOSSBilling\Exception('Can not remove group with clients');
+            throw new \FOSSBilling\Exception('Cannot remove groups with clients');
```

Comment:
It's technically only one group, I just think it sounds better to have groups plural since clients is as well.
Personally I think it would be fine to put either "Cannot remove groups with clients" or "Cannot remove a group with clients".

(Or even "Cannot remove a group with assigned clients" now that I am thinking about it)